src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsTable/DataTypeCell.tsx

```diff
@@ -40,7 +40,9 @@ const DataTypeCell = (props: { element: DataElement; codeSystemMap: any }) => {
         </span>
       </div>
       <div className="element-type">
-        {_.startCase(element?._type.split("QDM::")[1])}
+        {element?.description
+          ? element.description.substring(element.qdmTitle.length + 2)
```

Comment:
Part of a condition in case it's empty, can change if unneeded

Reply:
you probably don't need else part of the condition`(_.startCase(element?._type.split("QDM::")[1])})`. it doesn't add any value by displaying data type again. I would just display element name if present.

`{element?.description && element.description.substring(element.qdmTitle.`length + 2)}